embark-ui/src/components/ContractsDeployment.js

```diff
@@ -249,6 +249,9 @@ ContractsHeader.propTypes = {
 };
 
 const Contract = ({web3, contract, deploymentPipeline, web3Deploy, web3EstimateGas, web3Deployments, web3GasEstimates, toggleContractOverview}) => {
+  if (!contract.code && !contract.deploy) {
+    return <React.Fragment/>;
+  }
```

Comment:
I believe there is checks into EmbarkContract and Web3Contract components that prevent to display button/action in case the contract is an interface.

I think we could remove those checks

L188: `<span>{(contract.address && `deployed at ${contract.address}`) || 'not deployed'}</span>`
L125: `if (!constructor) return false;`

There might be more

Reply:
For L188, I don't think it's the same. Not being deployed doesn't mean it's an interface. 
For L125, not sure is no constructor === interface, but keeping that condition is just a safeguard that I think we could keep.

Reply:
👍